test/test_web.rb

```diff
@@ -647,12 +641,6 @@ def perform(a, b)
       assert_equal 200, last_response.status
       assert_match(/#{params.first['args'][2]}/, last_response.body)
     end
-
-    it 'handles bad query input' do
```

Comment:
This test depended on the link generated for the live poll button being present in the response body. Rewritten and moved to `test/test_web_helpers.rb`.